migrations/util/timescaleDB.js

```diff
@@ -30,13 +30,15 @@ function createContinuousAggregate ({
   interval,
   timeBucketLabel,
   column = 'createdTimestamp',
+  secondaryColumn
```

Comment:
Don’t you think we may add other columns in the future?
You’re probably right if you don’t, as having `secondaryColumns` would probably break the YAGNI rule. But it never hurts to plan a bit and explain your choices.

We need to ensure compressed rows have enough data in TimescaleDB and indeed there shouldn’t be too many dimensions to keep effective compression. You should probably specify this in your commit message, or comment TimescaleDB code.

Reply:
I considered passing an array of columns but kept only one secondary column because the number of dimensions will quickly increase the number of rows if the different dimensions have many possible values.

I'll add an explanation for this choice, thanks.